src/main/java/com/idrsolutions/microservice/storage/BaseStorage.java

```diff
@@ -10,10 +10,7 @@
  */
 public abstract class BaseStorage implements Storage {
     protected static final Logger LOG = Logger.getLogger(BaseStorage.class.getName());
-
-    /**
-     * @inheritDoc
-     */
+    
```

Comment:
Rather than delete, couldn't this be updated to:

```suggestion

    /**
     * {@inheritDoc}
     */
```

?